userland/tools/flash/storm-flash-app.py

```diff
@@ -34,6 +34,7 @@
 
 # create jlink script
 with open(build_dir+'/flash-app.jtag', 'w') as jlink_file:
+    jlink_file.write('eoe 1\n');
```

Comment:
@alevy this is not used anymore right?

Reply:
Oh yeah, I believe it isn't. `storm` is dead.